server/index.js

```diff
@@ -2,10 +2,33 @@ var express = require('express');
 var bodyParser = require('body-parser');
 var cors = require('cors');
 var path = require('path');
+var db = require('../database/db.js');
 
 
 var app = express();
-app.use(express.static(__dirname + '/../client/dist'));
+app.use('/:id', express.static(__dirname + '/../client/dist'));
 
-console.log('listening on port 3000');
-app.listen(3000);
\ No newline at end of file
+app.get('/:id/minNightlyRate', (req, res) => {
+
+  db.getMinNightlyRate(req.params.id, (rate) => {
+    res.status(200);
+    res.send({minNightlyRate: rate.minRate});
+    res.end();
+  });
+});
+
+app.get('/:id/availableDates', (req, res) => {
+
+  db.getAvailableDates(req.params.id, (dates) => {
+    res.status(200);
+    res.send(dates);
+    res.end;
+  });
+});
+
+
+
+console.log('listening on port 5001');
```

Comment:
This is great!
A very small suggestion (like really really small) below:

- An alternative way of managing ports is to declare a variable.
- `const port = 5001`
- This may help in readability and making change at only one location when updating port value. (console log and listen)

Note: This is, I think, only a coding style. May not necessarily be a standard but I though might be helpful 😃